### DISCUSSION TITLE
View all transactions missing latest transaction

### DISCUSSION THREAD

**Discussion by ghost** [blockscout/blockscout#8753] [Q&A]
Hi, I am using the blockscout with my private network (POA). 
My network:
- ~14M block
- > 163,444 transations
- interval 5
Now, when I go to Blockscout overview I can see the latest transaction, but when I go to `View All Transactions`, I don't see the latest transaction. I don't think it is a bug, but can you check for me?

![Screenshot 2023-10-31 at 07 50 11](https://github.com/blockscout/blockscout/assets/43759289/8a6d5c2f-95ae-4e58-8ae3-d95dbffe7976)
![Screenshot 2023-10-31 at 07 40 26](https://github.com/blockscout/blockscout/assets/43759289/279483a6-de68-4536-9f2e-d4c89bcd626a)

**Comment by ekawibisono** [comment]
Does the block run according to your private node? and then what client node do you use?

**Reply by ghost** [reply]
Yes, Blockscout is connecting to an archive node.

**Reply by ekawibisono** [reply]
You using geth ethereum ?

**Reply by ghost** [reply]
Yes

**Reply by captmilex** [reply]
I made a transaction on PGN network L2-L1 its 12days now and am yet to recieve my ETH. Please advice

**Reply by ekawibisono** [reply]
> Yes

Can you show you run geth ethereum ?
